src/app/board/board.component.ts

```diff
@@ -6,8 +6,8 @@ import { WorkItemService } from '../work-item/work-item.service';
 
 @Component({
   selector: 'my-board',
-  templateUrl: '/board.component.html',
-  styleUrls: ['/board.component.scss']
+  templateUrl: './board.component.html',
+  styleUrls: ['./board.component.scss']
```

Comment:
Are these failing for the missing '.' after update? I bet './' is the standard way.